### ISSUE TITLE
the difference of vel_world in rollout_global_motion and reset_root_velocity

### ISSUE THREAD

**Issue by rrenle** [yohanshin/WHAM#89]
Hello, Thanks your wonderful job. Now I have a question, Can you help me?
when computing the vel_world in rollout_global_motion, vel_world = (root[:, :-1] @ root_v.unsqueeze(-1)).squeeze(-1);
but in reset_root_velocity, vel_world = (poses_root[:, 1:] @ pred_vel.clone().detach().unsqueeze(-1)).squeeze(-1);
The former is root[:, :-1], the latter is poses_root[:, 1:]. Why do this?

**Comment by yohanshin** [comment]
Hi @rrenle , thank you for pointing this out. I think has been raised in the other issue. I think it's a mistake and I need to modify code for the consistency (although the results would not change largely given the small `dt` between frames.). But thanks for your point!
